### PR TITLE
Added some more Devil Design filaments

### PR DESCRIPTION
**Pull request by m-klecka**

### CODE REVIEW

**Comment by YanceyA** [comment]
Suggest that for the TPU that the hardness be amended. Such as TPU -> TPU-90A or TPU-95A.

**Comment by m-klecka** [comment]
> Suggest that for the TPU that the hardness be amended. Such as TPU -> TPU-90A or TPU-95A.

Makes sense. Done:)

File: filaments/devildesign.json [hunk 1]
```diff
@@ -463,6 +463,228 @@
                     "hex": "005387"
                 }
             ]
+        },
+        {
+            "name": "{color_name}",
+            "material": "SILK",
```
Comment (dresende) [inline filaments/devildesign.json:469]: Material should be `"PLA"` and add a property `"finish"` with value `"glossy"`.

File: filaments/devildesign.json [hunk 2]
```diff
@@ -463,6 +463,228 @@
...
+                },
+                {
+                    "name": "Bronze",
+                    "hex": "755847"
+                }
+            ]
+        },
+        {
+            "name": "{color_name}",
+            "material": "ABS+",
```
Comment (dresende) [inline filaments/devildesign.json:535]: I believe ABS+ is just ABS with perhaps a different density.

Comment (m-klecka) [reply filaments/devildesign.json:535]: Its a different filament. ABS with some additives or something like that. PLA and PLA+ are also different.

File: filaments/devildesign.json (hunk 1 above)
Comment (m-klecka) [reply filaments/devildesign.json:469]: Added

File: filaments/devildesign.json (hunk 2 above)
Comment (StuSerious) [reply filaments/devildesign.json:536]: with the new changes introduced in #21 ABS+ is allowed and should pass checks, as its indeed a different material.

**Comment by StuSerious** [comment]
@m-klecka did you commit b7e5a0f77d5b4619be5ad50c152441203d042f6c because of failing checks?

**Comment by m-klecka** [comment]
> @m-klecka did you commit b7e5a0f77d5b4619be5ad50c152441203d042f6c because of failing checks?

No. I think it makes more sense to just have TPU, since the hardness is not a name of the material, rather just it's property.

**Comment by StuSerious** [comment]
I think we should keep it in there, it's easy enough for any client to deal with and it reflects the actual shore hardness of the material in question. I fixed it in 21 so that it applies and passes schema validation.

Anyway let's wait to hear what Donkie says.

File: filaments/devildesign.json (hunk 2 above)
Comment (YanceyA) [reply filaments/devildesign.json:536]: Agreed, ABS and ABS+ should be differentiated if possible for no other reason as that is how the market names them. ABS+ is ABS with additional additives (which may include pother polymers). More marketing than science here.

**Comment by Donkie** [comment]
I think it's good to have it in the material name if the manufacturer states it, better to have more info than less

**Comment by m-klecka** [comment]
@Donkie anything else i should do/edit to get this merged?

**Comment by Donkie** [comment]
No sorry, just been busy with other things so haven't had time to look at this.

Thanks for the contribution :)

**Pull request merged**